README.md

```diff
@@ -1,61 +1,49 @@
-- Automatic note creation for the [Anki](https://apps.ankiweb.net/) flashcard program via the [AnkiConnect](https://foosoft.net/projects/anki-connect) plugin.
-- Clean, modern code makes it easy for developers to [contribute](https://github.com/themoeway/yomitan/blob/master/CONTRIBUTING.md) new features.
+- 📈 Interactive popup definition window for displaying search results.
+- 🔊 Built-in audio playback for select Japanese words.
+- ✍ Kanji stroke order diagrams are generally just a click away.
+- 📝 [Automatic note creation](./docs/anki-integration.md#anki-integration) for the [Anki](https://apps.ankiweb.net/) flashcard program via the [AnkiConnect](https://foosoft.net/projects/anki-connect) plugin.
+- 🔍 Custom search page for easily executing custom search queries.
+- 📖 Support for multiple dictionary formats including [EPWING](https://ja.wikipedia.org/wiki/EPWING) via the [Yomitan Import](https://github.com/themoeway/yomitan-import) tool.
+- ✨ Clean, modern code makes it easy for developers to [contribute](#contributing) new features.
 
 [![Term definitions](img/ss-terms-thumb.png)](img/ss-terms.png)
 [![Kanji information](img/ss-kanji-thumb.png)](img/ss-kanji.png)
 [![Dictionary options](img/ss-dictionaries-thumb.png)](img/ss-dictionaries.png)
 [![Anki options](img/ss-anki-thumb.png)](img/ss-anki.png)
 
-## Helpful information<!-- omit from toc -->
+## Helpful information
 
 - [Migrating from Yomichan (legacy)](./docs/yomichan-migration.md#migrating-from-yomichan)
-- [Importing standardised and custom dictionaries](./docs/dictionaries.md#dictionaries)
-- [Anki integration and flashcards creation](./docs/anki-integration.md#anki-integration)
-- [Advanced options, including MeCab](./docs/advanced-options.md#advanced-options)
 - [Frequently asked questions](./docs/faq.md#frequently-asked-questions)
+- [Recommended and custom dictionary import guide](./docs/dictionaries.md#dictionaries)
+- [Anki integration and flashcards creation guide](./docs/anki-integration.md#anki-integration)
 - [Keyboard shortcuts](./docs/keyboard-shortcuts.md)
+- [Advanced options, including MeCab](./docs/advanced-options.md#advanced-options)
 
 ## Installation
 
-Yomitan comes in two flavors: _stable_ and _testing_. New changes are initially introduced into the _testing_ version, and after some time spent ensuring that they are relatively bug free, they will be promoted to the _stable_ version. If you are technically savvy and don't mind submitting issues on GitHub, try the _testing_ version; otherwise, the _stable_ version will be your best bet.
+Yomitan comes in two flavors: _stable_ and _testing_. New changes are initially introduced into the _testing_ version, and after some time spent ensuring that they are relatively bug free, they will be promoted to the _stable_ version. If you are technically savvy and don't mind [submitting issues](#contributing) on GitHub, try the _testing_ version; otherwise, the _stable_ version will be your best bet.
```

Comment:
```suggestion
Yomitan comes in two flavors: _stable_ and _testing_. New changes are initially introduced into the _testing_ version, and after some time spent ensuring that they are relatively bug free, they will be promoted to the _stable_ version. If you are technically savvy and don't mind [submitting issues](https://github.com/themoeway/yomitan/issues/new/choose) on GitHub, try the _testing_ version; otherwise, the _stable_ version will be your best bet. Check [contributing](#contributing) for more information on how to help.
```